src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -28,6 +28,8 @@
 class VideoLTIViewTestCase(TestCase):
     """Test the video view in the ``core`` app of the Marsha project."""
 
+    maxDiff = None
```

Comment:
@sampaccoud for debugging purpose I would like to keep it. The diffs in this class are huge and it's always a nightmare to understang what is failing